### PR TITLE
OCLOMRS-265: Change the versions table heading to say “Version(s)” instead of “Released Version”

### PR DESCRIPTION
**Pull request by Duncanian**
# JIRA TICKET NAME:
[Change the versions table heading to say “Version(s)” instead of “Released Version”](https://issues.openmrs.org/browse/OCLOMRS-265)

# Summary:
On a dictionary details page, the versions table heading should be “Version(s)” instead of “Released Version”.

### CODE REVIEW

**Comment by Michyjones** [comment]
@Duncanian Fix the test the build  failing

File: src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx [hunk 1]
```diff
@@ -198,7 +198,6 @@ const DictionaryDetailCard = (props) => {
         handleChange={handleChange}
         inputLength={inputLength}
       />
-
```
Comment (dondrzzy) [inline src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: Is this a formatting change?

Comment (Duncanian) [reply src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: Yeah, it's a formatting change.

Comment (dondrzzy) [reply src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: Can this be done in a different PR please?

Comment (Michyjones) [reply src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: +1 on this

File: src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx [hunk 2]
```diff
@@ -154,7 +154,7 @@ const DictionaryDetailCard = (props) => {
             </ul>
           </fieldset>
           <fieldset>
-            <legend>Released Version</legend>
+            <legend>{releasedVersions.length > 1 ? 'Versions' : 'Version'}</legend>
```
Comment (dkayiwa) [inline src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: Do you have tests for both "Version" and "Versions"?

Comment (Duncanian) [reply src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: Yeah, they have both been tested.

Comment (dkayiwa) [reply src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: Can you point me to each?

File: src/tests/Dictionary/DictionaryContainer.test.jsx [hunk 3]
```diff
@@ -144,6 +144,31 @@ describe('DictionaryOverview', () => {
...
+      fetchDictionaryConcepts: jest.fn(),
+    };
+    const wrapper = mount(<Provider store={store}>
+      <MemoryRouter>
+        <DictionaryOverview {...props} />
+      </MemoryRouter>
+    </Provider>);
+    expect(wrapper).toMatchSnapshot();
+  });
+
```
Comment (Duncanian) [inline src/tests/Dictionary/DictionaryContainer.test.jsx]: @dkayiwa , here is the test that covers both of them.

File: src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx (hunk 2 above)
Comment (Duncanian) [reply src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx]: @dkayiwa , I have pointed it out.

**Pull request merged**